js/jquery.pickmeup.js

```diff
@@ -681,12 +699,19 @@
 						options.binded.update_date();
 					}
 					// Move current month to the first place
-					options.current.addMonths(Math.floor(options.calendars / 2) - instance_index);
+					options.current.addMonths(align(options) - instance_index);
 				} else {
 					var val	= parseInt(el.text(), 10);
-					options.current.addMonths(instance_index - Math.floor(options.calendars / 2));
+					options.current.addMonths(instance_index - align(options));
 					if (el.hasClass('pmu-not-in-month')) {
 						options.current.addMonths(val > 15 ? -1 : 1);
+						if (options.alignment === parseInt(options.alignment, 10)) {
+							instance_index = instance_index + (val > 15 ? -1 : 1);
+							options.alignment = instance_index;
+						}
+					}
+					if (options.alignment === parseInt(options.alignment, 10)) {
+						options.alignment = instance_index;
```

Comment:
What about https://api.jquery.com/jQuery.isNumeric/ instead of `options.alignment === parseInt(options.alignment, 10)`?
Since we already depend on jQuery, this would require less brain power to understand what is going on)